export/patterns/04-pages/40-portfolio-main.mustache

```diff
@@ -2,14 +2,8 @@
 	{{> templates-header}}
 	{{> molecules-secondary-nav}}
 
-	{{#portfolio-story-exp}}
-		{{> 02-organisms/40-portfolio/11-portfolio-storytelling}}
-	{{/portfolio-story-exp}}
-
 	<div class="row">
 		{{> 01-molecules/05-navigation/03-tertiary-nav}}
 		{{> templates-portfolio-main}}
 	</div>
 </div>
-
-{{> organisms-footer}}
```

Comment:
Should this be removed? Was it unused or will it remove the footer?

Reply:
Didn't see that, sorry! it should be fixed now